r-package/grf/NAMESPACE

```diff
@@ -31,5 +31,6 @@ export(split_frequencies)
 export(test_calibration)
 export(tune_causal_forest)
 export(tune_ll_regression_forest)
+export(tune_ll_causal_forest)
```

Comment:
These should be alphabetized, unfortunately there's an issue with the build and we need to do it by hand :(

Reply:
I just fixed the issue in #423, so now you shouldn't need to touch the `NAMESPACE` file at all!